stable/atlantis/values.yaml

```diff
@@ -203,3 +203,29 @@ environmentSecrets: []
 #     secretKeyRef:
 #       name: the_k8s_secret_name
 #       value: the_key_of_the_value_in_the_secret
+
+# Optionally specify google service account credentials as Kubernetes secrets.
+#
+# The Kubernetes secret can be created from a google service account JSON file using:
+#
+#   kubectl create secret generic the_k8s_secret_name --from-file=key.json=PATH-TO-KEY-FILE.json
+#
+# Once included here you must update your terraform google provider configuration to include:
```

Comment:
I don't think this should be in the docs for the helm chart. Not everyone will be using Terraform and this will confuse them.